proto/publicrpc/v1/publicrpc.proto

```diff
@@ -77,6 +83,41 @@ message GetSignedVAAResponse {
   bytes vaa_bytes = 1;
 }
 
+message GetNonGovernanceVAABatchRequest {
+  // Emitter chain ID.
+  ChainID emitter_chain = 1;
+  // Hex-encoded (without leading 0x) emitter address.
+  string emitter_address = 2;
+  // Target chain ID.
+  ChainID target_chain = 3;
+  // Sequences
+  repeated uint64 sequences = 4;
+}
+
+message GetNonGovernanceVAABatchResponse {
+  message Entry {
+    uint64 sequence = 1;
+    bytes vaa_bytes = 2;
+  }
+
+  repeated Entry entries = 1;
+}
+
+message GetGovernanceVAABatchRequest {
+  // Sequences
+  repeated uint64 sequences = 1;
+}
+
+message GetGovernanceVAABatchResponse {
+  message Entry {
+    ChainID target_chain = 1;
+    uint64 sequence = 2;
+    bytes vaa_bytes = 3;
+  }
+
+  repeated Entry entries = 1;
+}
+
```

Comment:
Why do we need these new endpoints (not from WH)?

Reply:
This is used by the explorer-backend to sync missing VAAs from the guardian. The wormhole explorer-backend currently does not support syncing missing VAAs.